### PR TITLE
feat: use rollup and babel for building

### PR DESCRIPTION
**Pull request by gkatsev**
This is my proposal for a solution to #7.
It uses rollup to build the output, it will output a cjs, es, and umd file. We probably don't even need separate cjs and es files because there aren't any dependencies, but I kept it around for now just in case.
The benefit of rollup is that due to this being a library and rollup's treeshaking, we get rid of a lot of the boilerplate removed from using something like browserify for building the UMD file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
+    "rollup": "^0.55.1",
+    "rollup-plugin-babel": "^3.0.3",
     "standard": "^10.0.3",
     "watchify": "^3.9.0"
   },
   "scripts": {
-    "build": "browserify lib/index.js > build/can-autoplay.js --standalone canAutoplay && google-closure-compiler-js 2>/dev/null build/can-autoplay.js > build/can-autoplay.min.js --languageOut=ES6",
-    "watch": "watchify lib/index.js -o build/can-autoplay.js -v --standalone canAutoplay",
+    "build": "rollup -c",
+    "minify": "google-closure-compiler-js 2>/dev/null build/can-autoplay.js > build/can-autoplay.min.js --languageOut=ES6",
```
Comment (gkatsev) [inline package.json]: I kept the minify as a separate command. How are released done for this project? Is having the build file in the repo something we want to keep? Personally, I prefer not to check in built files into the repo and just have them published to npm.

File: package.json [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
     "lint": "standard --fix 'lib/**/*.js' 'test/**/*.js'",
     "test": "ava"
   },
-  "dependencies": {},
   "ava": {
     "files": [
       "test/*.spec.js"
     ],
     "require": [
+      "babel-register",
```
Comment (gkatsev) [inline package.json]: this is required for ava to handle ES6 imports properly.

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import babel from 'rollup-plugin-babel';
```
Comment (gkatsev) [inline rollup.config.js]: this is a pretty standard and minimal rollup config.

**Comment by gkatsev** [comment]
Also, I guess I would need to update the `yarn.lock` file for this PR.

File: package.json (hunk 1 above)
Comment (NicolasSiver) [reply package.json]: I assume watchify and browserify can be removed.
Rollup can not build minified versions? Like `webpack -p` ?

File: package.json [hunk 4]
```diff
@@ -1,30 +1,37 @@
 {
   "name": "can-autoplay",
   "version": "2.0.1",
-  "main": "lib/index.js",
+  "main": "build/can-autoplay.cjs.js",
+  "module": "build/can-autoplay.es.js",
+  "browser": "build/can-autoplay.js",
```
Comment (NicolasSiver) [inline package.json]: what will use `browser` field?

File: package.json (hunk 1 above)
Comment (gkatsev) [reply package.json]: oh, yeah, I forgot that we don't need watchify and browserify anymore.
Rollup can minify via a plugin. I can change it to have rollup minify instead.

File: package.json (hunk 4 above)
Comment (gkatsev) [reply package.json]: It can probably be removed. I copied this from a rollup example file.

**Pull request merged**